BUILD.ndk_root.tpl

```diff
@@ -36,3 +36,17 @@ cc_library(
     ]),
     linkopts = ["-ldl"],
 )
+
+cc_library(
```

Comment:
Let's add a note saying that new code should use GameActivity: https://developer.android.com/games/agdk/game-activity